generated/billing/log.out

```diff
@@ -186,327 +186,331 @@ warning | PreCheck/DuplicateSchema | Duplicate Schema named 'ErrorResponse' (1 d
 warning | PreNamer/DeduplicateName | Deduplicating schema name: 'EnrollmentAccountListResult' -> 'EnrollmentAccountListResultAutoGenerated'
 warning | PreNamer/DeduplicateName | Deduplicating schema name: 'ErrorResponse' -> 'ErrorResponseAutoGenerated'
 warning | PreNamer/DeduplicateName | Deduplicating schema name: 'ErrorDetails' -> 'ErrorDetailsAutoGenerated'
-info    | autorest.bicep took 1ms
+warning | Skipping resource type microsoft.billing/invoices under path '/subscriptions/{subscriptionId}/providers/Microsoft.Billing/invoices/{invoiceName}': Found multiple definitions for the same type
```

Comment:
@jeskew  See what caused this and update this comment with an explanation.

Reply:
This may have been caused by the existence of `/subscriptions/{subscriptionId}/providers/Microsoft.Billing/invoices/{invoiceName}` alongside a `/subscriptions/{subscriptionId}/providers/Microsoft.Billing/invoices/latest` endpoint, both of which return an invoice resource. That pattern (using a non-parameterized path segment as a pointer to a resource with a certain property (but not one with a fixed ID)) may be more common with read-only resources.